packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -116,10 +115,10 @@
 def get_single_track(track_id, current_user_id, endpoint_ns, exclude_gated=True):
     args = {
         "id": [track_id],
-        "with_users": True,
         "filter_deleted": False,
         "exclude_gated": exclude_gated,
         "current_user_id": current_user_id,
+        "skip_unlisted_filter": True,
```

Comment:
nit: how about `show_unlisted`?

Reply:
I think that implies setting it to false will _not_ show unlisted tracks, which isn't necessarily true

Reply:
oh i see you just removed an arg with that name.. is it dangerous to re-use the name?

Reply:
not dangerous to reuse, but they mean different things